sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/__init__.py

```diff
@@ -16,16 +16,19 @@
     FeatureFlagConfigurationSetting,
     SecretReferenceConfigurationSetting,
 )
+from ._generated.models import Snapshot, ConfigurationSettingFilter
 from ._version import VERSION
 from ._azure_appconfiguration_error import ResourceReadOnlyError
 
 __version__ = VERSION
 __all__ = [
     "AzureAppConfigurationClient",
     "ConfigurationSetting",
+    "ConfigurationSettingFilter",
     "ResourceReadOnlyError",
     "FeatureFlagConfigurationSetting",
     "SecretReferenceConfigurationSetting",
+    "Snapshot",
```

Comment:
@annatisch it looks to me `Snapshot` is a return-only type and users don't need to create a `Snapshot` object. In this case, do we need to expose `Snapshot`?

Reply:
Btw, my understanding is that we should make all custom types - including input types and output types - in api surface to be public,